plugin/pkg/scheduler/schedulercache/pods_to_cache.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schedulercache
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// PodsToCache is used for testing
+type PodsToCache []*api.Pod
```

Comment:
PodsToCache should be used only for testing purposes, I think.
Can you:
- move it to testing package
- change the name so that it's more obvious that it is a testing type (e.g. TestCache or sth like that)?